packages/react-charting/src/utilities/ChartHoverCard/ChartHoverCard.styles.ts

```diff
@@ -1,83 +1,71 @@
-import { IChartHoverCardStyles, IChartHoverCardStyleProps } from './ChartHoverCard.types';
-import { FontWeights } from '@fluentui/react/lib/Styling';
+import { IChartHoverCardStyles, IChartHoverCardStyleProps, ChartHoverCardVariant } from './ChartHoverCard.types';
+import { FontSizes, FontWeights } from '@fluentui/react/lib/Styling';
 
 export const getChartHoverCardStyles = (props: IChartHoverCardStyleProps): IChartHoverCardStyles => {
-  const { color, XValue, theme, isRatioPresent = false } = props;
+  const { theme, variant, hasBothMetrics } = props;
   return {
-    calloutContentRoot: [
-      {
-        display: 'grid',
-        overflow: 'hidden',
-        padding: '11px 16px 10px 16px',
-        backgroundColor: theme.semanticColors.bodyBackground,
-        backgroundBlendMode: 'normal, luminosity',
-      },
-    ],
+    calloutContentRoot: {
```

Comment:
Just found out that single callout is showing the left color border as white in high contrast mode. Because the forcedColorAdjust high contrast selector is not set.
But it is set for multi value callout.